package_manager/cloudbuild.yaml

```diff
@@ -1,30 +0,0 @@
-# A cloud build config to release a PAR binary of
-# the dpkg_parser.par used for the package manager bazel rules
-
-# A cloudbuild is triggered by every commit and $COMMIT_SHA is a built-in 
-# substitution
-
-steps:
-# Build the dpkg_parser PAR file
-# this binary is used by the package manager rules
-# to download debian package lists and debian packages
-- name: gcr.io/cloud-builders/bazel
-  args: [
-    '--output_base', '/workspace',
-    'build', '//package_manager:dpkg_parser.par',
-    # TODO(r2d4): Remove once PAR compilation runs properly inside
-    # the Bazel sandbox on cloudbuild.
-    '--strategy', 'PythonCompile=standalone'
-  ]
-  env: ['PROJECT_ID=$PROJECT_ID']
-
-# Upload the dpkg_parser PAR file to a GCS bucket
-- name: gcr.io/cloud-builders/gsutil
-  args: [
-    'cp',
-    'bazel-bin/package_manager/dpkg_parser.par',
-    'gs://distroless/package_manager_tools/$COMMIT_SHA/dpkg_parser.par'
```

Comment:
Don't we still need to publish this .par, since others are using it?

Reply:
They can keep using the existing ones. No more new uploads from now–another intended side-effect of this change. If someone really wants a new binary, anyone can always build from source.